features/customized-built-in-elements.yml

```diff
@@ -0,0 +1,9 @@
+name: Customized built-in elements
+description: Customized built-in elements are HTML elements with behavior that you define.
```

Comment:
This description should mention the `is` attribute.

Reply:
How about …

```suggestion
description: Customized built-in elements are HTML elements that extend built-in elements using the `is` attribute, to add new behaviors that you define.
```

Reply:
Thank you!